### PR TITLE
Add documentation regarding thread creation logic

### PR DESCRIPTION
**Pull request by benfoster04**
Based off information in #30.

### CODE REVIEW

**Review by kk5dire — APPROVED** [review]
LGTM

File: main_utils.py [hunk 1]
```diff
@@ -25,6 +25,10 @@ async def on_channel_create(self, channel):
         """
         EXTENSION 1 + 3: Staff notes for tickets! and button stripping.
         Minor Extension 2: Safe Community Support
+        
+        This event works based on the ticket_users configuration.
+        Please ensure you add your ticket bot ID or approved
+         ticket invoking user IDs to the config file or via commands.
```
Comment (Tech-TTGames) [inline main_utils.py:31]: FOR FECKS SAKE FORMATTING

File: main_utils.py [hunk 2]
```diff
@@ -25,6 +25,10 @@ async def on_channel_create(self, channel):
         """
         EXTENSION 1 + 3: Staff notes for tickets! and button stripping.
         Minor Extension 2: Safe Community Support
+        
+        This event works based on the ticket_users configuration.
+        Please ensure you add your ticket bot ID or approved
```
Comment (Tech-TTGames) [inline main_utils.py:30]: Capitialize Tickets

**Review by Tech-TTGames — CHANGES_REQUESTED** [review]

File: main_utils.py (hunk 1 above)
Comment (benfoster04) [reply main_utils.py:31]: THE STUPID FECKING DEEPSOURCE DOESNT LIKE HOW LONG OF A SENTENCE IT IS SO I INDENTED IT WHAT DO YOU WANT FROM ME HJKDKASKDH

Comment (Tech-TTGames) [reply main_utils.py]: THE SPACE

Comment (kk5dire) [reply main_utils.py]: > THE STUPID FECKING DEEPSOURCE DOESNT LIKE HOW LONG OF A SENTENCE IT IS SO I INDENTED IT WHAT DO YOU WANT FROM ME HJKDKASKDH
You could at least allign the lines :(

Comment (Tech-TTGames) [reply main_utils.py]: AT THE BEGGINING

Comment (Tech-TTGames) [reply main_utils.py]: *points gun*

Comment (benfoster04) [reply main_utils.py]: I did it BECAUSE IT WAS TOO LONG AND SHOWS ITS A CONTINUATION SUGMANUTS

Comment (benfoster04) [reply main_utils.py]: fine I'll change it but don't say I didn't try smh

Comment (kk5dire) [reply main_utils.py]: @benfoster04 please, you are scaring the children with your dangling space

**Review by Tech-TTGames — APPROVED** [review]
LGTM

**Pull request merged**